act-rules-format.bs

```diff
@@ -235,158 +257,198 @@ When all expectations are true for a test target, the test target `passed` the r
 
 
-ACT Data Format (Output Data) {#output}
-=======================================
-
-With ACT Rules, it is important that data coming from different sources can be compared. By having a shared vocabulary, accessibility data that is produced by different auditors can be compared and, where necessary, aggregated. Therefore, every ACT Rule MUST express the output in a format that has all of the features described in the ACT Data Format.
-
-Rules are tested in two steps. Firstly, the applicability is used to find a list of [Test Targets](#output-test-target) (elements, tags or other "components") within the web page or other [test subject](#output-test-subject). Then each test target is tested to see if all of the [expectations](#expectations) are true. This will give the outcome for each test target. For contextual information, the output data must also include [test subject](#output-test-subject) and the [rule identifier](#rule-identifier).
-
-This will mean that every time a rule is executed on a page, it will return a set with zero or more results, each of which MUST have at least the following properties:
+Evaluation Instructions {#evaluation-instructions}
+========================
 
-- [Rule Identifier](#rule-identifier) (test)
-- [Test Subject](#output-test-subject) (Web page)
-- [Test Target](#output-test-target) (pointer)
-- [Outcome](#output-outcome) (`Passed`, `Failed`, or `Inapplicable`)
+An ACT Rule MUST include evaluation instructions. Evaluation instructions describes how applicability and expecations can be used to determine (non-)conformance to the [listed accessibility requirements](#accessibility-requirements). Evaluation instructions MUST indicate how `passed`, `failed` and `inapplicable` [=outcomes=] can be determined for each [=test subject=], when evaluating the rule. 
 
 <div class=example>
-Output data using EARL and JSON-LD. (See [Evaluation and Report Language](https://www.w3.org/WAI/standards-guidelines/earl/) [[EARL10-Schema]] and [Java Script Object Notation (JSON)](https://www.json.org).)
-
-```javascript
-{
-  "@context": "https://raw.githubusercontent.com/w3c/wcag-act/master/earl-act.json",
-  "@type": "Assertion",
-  "subject": "https://example.org/",
-  "test": "auto-wcag:rules/SC1-1-1-css-image.html",
-  "result": {
-    "outcome": "Failed",
-    "pointer": "html > body > h1:first-child"
-  }
-}
-```
+  <ol>
+    <li>If no parts of the test subject match the applicability, the outcome is `inapplicable`</li>
+    <li>Otherwise, if all expectations of the rule are true for each test target, the outcome is `passed`.</li>
+    <li>Otherwise the outcome is `failed`.<li>
 </div>
 
+If any [accessibility requirements](#accessibility-requirements) are listed, the ACT Rule MUST indicate how each outcome impacts the conformance to the listed accessibility requirements. For WCAG 2.1 Success Criteria, this means indicating whether or not the success criterion can be satisfied, or if further testing is needed.
```

Comment:
"impacts" -> "relates to"

Replace the second sentence with the following:
"Typically failing a rule means the associated accessibility requirement was not met. At the same time, passing rule does not usually mean that the associated accessibility requirement was met, unless the rule was designed to address all fail and pass conditions of that accessibility requirement."